react/features/toolbox/components/web/OverflowMenuProfileItem.js

```diff
@@ -80,7 +75,9 @@ class OverflowMenuProfileItem extends Component<Props> {
                 className = { classNames }
                 onClick = { this._onClick }>
                 <span className = 'overflow-menu-item-icon'>
-                    <Avatar uri = { avatarURL } />
+                    <Avatar
+                        participantId = { _localParticipant.id }
+                        size = { 24 } />
```

Comment:
Extract constant to styles

Reply:
But that's web. I don't want to start using styles.js on web. Or did you mean something else?